src/types.rs

```diff
@@ -18,7 +18,7 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 //! Provides Astarte specific types to be used by the
-//! [AstarteDeviceSdk][crate::AstarteDeviceSdk] to transmit/receivedata to/from the Astarte cluster.
+//! [AstarteDeviceSdk][crate::AstarteDeviceSdk] to transmit/receive data to/from the Astarte cluster.
```

Comment:
I would remove data altogether

Reply:
Do you mean maintaining only `receive` without `data`?